docs/src/concepts/bc.md

```diff
@@ -0,0 +1,46 @@
+# Boundary Conditions
```

Comment:
General comment for BCs, as you can see in file https://github.com/PTsolvers/Chmy.jl/blob/main/examples/batcher.jl, one can either precompute BCs, or apply them on-the-fly; maybe worth mentioning. Also, one could mention that not specifying is the same as specifying `nothing` as seen in the examples.

Reply:
@youwuyou I would add an example showing how to precompute BCs or do the on-the-fly syntax as in some cases it could be useful